Makefile

```diff
@@ -537,6 +537,10 @@ verify-modules: modules
 		git diff; \
 		echo "go module files are out of date"; exit 1; \
 	fi
+	@if (find . -name 'go.mod' -print0 | xargs -n1 grep -q -i 'k8s.io/client-go.*+incompatible'); then \
```

Comment:
Do you need to escape the `+`?

Reply:
if I do, it says `grep: repetition-operator operand invalid`, so I guess no?

Reply:
Oh this isn't egrep, ok